.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "singleQuote": true
+}
```

Comment:
While I can't speak for the project, random dotfiles is a smell to me. AFAIK, this project doesn't use https://prettier.io/.

Reply:
Yeah, my bad - gonna kill it.